Orange/preprocess/__init__.py

```diff
@@ -6,10 +9,3 @@
 from .remove import *
 from .preprocess import *
 
-from Orange.misc.lazy_module import _LazyModule
-transformation = _LazyModule("preprocess.transformation")
-score = _LazyModule("preprocess.score")
-discretize = _LazyModule("preprocess.discretize")
-continuize = _LazyModule("preprocess.continuize")
-normalize = _LazyModule("preprocess.normalize")
-remove = _LazyModule("preprocess.remove")
```

Comment:
`preprocess` is already a lazy module itself, so there's no point in making these lazy.